app/views/graders/_graders_manager.js.jsx.erb

```diff
@@ -589,9 +595,9 @@
       }.bind(this));
 
       return (
-       <div className='tab-pane ui-tabs ui-widget ui-widget-content ui-corner-all' id='groups-tabs'>
-        <Table data={groups_data} columns={columns}
-               filters={filters} filter_type={false}
+        <div className='tab-pane ui-tabs ui-widget ui-widget-content ui-corner-all' id='groups-tabs'>
+          <Table data={groups_data} columns={columns} ref="myTable"
+               filters={filters} filter_type={filters.length>3}
```

Comment:
In the table.js file is suggest to use the select filter type when there's more than 3 choices to filter by.